pages/guide/07-full-stack-deployments.md

```diff
@@ -1,4 +1,156 @@
-> **Note**: this guide is a work in progress and is not currently complete. Content
-> here will change and be added over time. In the meantime, you can check out the
-> [Gleam Discord server](https://discord.gg/Fm8Pwmy) if you have any questions about
-> how to deploy full stack applications with Lustre.
+# 07 Full-Stack Deployments
+
+If you have built a full-stack Lustre application then you can use Docker and a platform provider such as [Fly.io](https://fly.io/) to deploy your application.
+
+In this guide, you will learn how to manually deploy a Lustre full-stack application to Fly.io using the `flyctl` CLI. You will also learn how to leverage GitHub Actions for continuous deployments.
+
+If you are planning on deploying a Lustre SPA without a backend, you probably don't want to follow this guide! Instead, refer to the [Lustre SPA deployment guide](./04-spa-deployments.html).
```

Comment:
`(./04-spa-deployments.html)`

is this how you correctly link to other pages? if not I need to update in a bunch of places